packages/button/README.md

```diff
@@ -47,24 +44,18 @@ Button labels tell users what will happen when they click the button. Use verbs
 - Delete \$object
 - Deploy, Scale, Restart
 
-Use sentence-style capitalization (only the first word in a phrase and any proper nouns capitalized) and no more than three words for button labels.
+Use sentence-style capitalization (only the first word in a phrase and any proper nouns capitalized). Include no more than three words for button labels.
 
-For Sets of Buttons, use specific labels, such as Save or Cancel, instead of using Yes and No. This is particularly helpful when the user is confirming an action.
+For sets of buttons, use descriptive action words in labels such as Save or Cancel instead of Yes and No. This is particularly helpful when the user is confirming an action.
```

Comment:
Sorry, this is super nitpicky, but do the commas you removed need to be there since the "such as" isn't technically part of a restrictive clause? Ignore me if I'm wrong. You would definitely know better than me, but thought I'd check!

Reply:
Grammarly didn't yell at me for it haha. It felt off to me the way it was originally when reading it. I'm going to refactor this into 2 sentences though.